sql/ddl_api.sql

```diff
@@ -213,3 +213,10 @@ CREATE OR REPLACE PROCEDURE refresh_continuous_aggregate(
     window_start             "any",
     window_end               "any"
 ) LANGUAGE C AS '@MODULE_PATHNAME@', 'ts_continuous_agg_refresh';
+
+-- Drop the specified chunk replica on the specified data node
+CREATE OR REPLACE FUNCTION  drop_chunk_replica(
```

Comment:
This file contains only public API. I suggest to move this function to `ddl_internal.sql`

Reply:
Ok, done